typescript/phoenix-sdk/src/utils/market.ts

```diff
@@ -360,10 +399,12 @@ function levelToUiLevel(
   quoteAtomsPerQuoteUnit: number
 ): [number, number] {
   return [
-    (toNum(priceInTicks) / quoteAtomsPerQuoteUnit) *
+    ((toNum(priceInTicks) / quoteAtomsPerQuoteUnit) *
       marketData.quoteLotsPerBaseUnitPerTick *
-      toNum(marketData.header.quoteLotSize),
-    toNum(sizeInBaseLots) / marketData.baseLotsPerBaseUnit,
+      toNum(marketData.header.quoteLotSize)) /
+      marketData.header.rawBaseUnitsPerBaseUnit,
+    (toNum(sizeInBaseLots) / marketData.baseLotsPerBaseUnit) *
```

Comment:
I think this is wrong:
`(BA / BL) / (BL / BU)` => `(BA * BU) / (BL * BL)`

Should be a multiplication